build-system/compilation-flags.cmake

```diff
@@ -1,10 +1,14 @@
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
 
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror")
 if (LIBIRM_BUILD_32_BITS)
   set (CMAKE_CXX_FLAGS -m32 ${CMAKE_CXX_FLAGS})
 endif()
 
-set (LIBIRM_CMAKE_CXX_FLAGS -fno-rtti)
-set (LIBIRM_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include)
\ No newline at end of file
+set (LIBIRM_CMAKE_CXX_FLAGS -fno-rtti -Wall -Werror)
```

Comment:
Removed these flags from the global config because of this problem https://github.com/llvm/llvm-project/commit/d0c8d58527b07d8609961ea1f084d69ce0fcd3a8